apps/web/src/modules/create-proposal/components/TransactionForm/CustomTransaction/forms/ABI/ABI.tsx

```diff
@@ -3,29 +3,24 @@ import { ethers } from 'ethers'
 import React from 'react'
 
 import { useCustomTransactionStore } from 'src/modules/create-proposal'
-import { useLayoutStore } from 'src/stores'
 import { AddressType } from 'src/typings'
 
 import { CustomTransactionForm } from '../CustomTransactionForm'
 import { fields, validateABI } from './fields'
 
 export const ABI = () => {
   const { customTransaction, composeCustomTransaction } = useCustomTransactionStore()
-  const { provider } = useLayoutStore()
   const initialValues = {
     transactionCustomABI: customTransaction?.customABI || '',
   }
 
-  const { signer } = useLayoutStore()
-
   const submitCallback = React.useCallback(
     (values: { transactionCustomABI: string }) => {
       try {
-        if (!!signer && !!customTransaction.address && !!values.transactionCustomABI) {
+        if (!!customTransaction.address && !!values.transactionCustomABI) {
```

Comment:
Why the double bang truthy tests here?

Reply:
removing this not sure why they are written like this either